docs/new-rule.md

```diff
@@ -24,5 +24,6 @@ Use the [`astexplorer` site](https://astexplorer.net) with the `espree` parser a
 	*(The description should be the same as the heading of the documentation file).*
 - Run `$ npm test` to ensure the tests pass.
 - Run `$ npm run integration` to run the rules against real projects to ensure your rule does not fail on real-world code.
+- Run `$ npm run lint` to run the rules against this repository to ensure code in the repository are following your rule.
```

Comment:
Why a separate step? It should just be part of `npm test`.

Reply:
I was thinking, when some one making a new rule, they can do this after their PR is reviewed, new rules might requires other code changes

Reply:
Good point. Ok.